### ISSUE TITLE
Dropdown: Accessibility issue

### ISSUE THREAD

**Issue by pratheeshpradeepkumars** [primefaces/primereact#4503]
### Describe the bug

When using accessibility check on Dropdown component its showing "Select missing label" alert.

![github](https://github.com/primefaces/primereact/assets/32036647/04042d80-d9a1-4d8d-b2e4-b79a7682c974)

Accessibility tool used: Wave - Chrome extension

### Reproducer

https://codesandbox.io/s/primereact-demo-forked-tt4nh2

### PrimeReact version

9.5.0

### React version

18.x

### Language

ES6

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. Install Wave chrome extension
2. Open https://codesandbox.io/s/primereact-demo-forked-tt4nh2
3. Open code preview in new window in codesandbox.
4. Activate wave tool in the browser.

![github2](https://github.com/primefaces/primereact/assets/32036647/64f09ef9-a634-4300-b9ce-2d6b8f1ba781)

![github](https://github.com/primefaces/primereact/assets/32036647/8fcabff7-e9f2-496e-911d-19fca20b757f)

**Comment by melloware** [comment]
That being said here is a Sanbox that removes that warning: https://codesandbox.io/s/primereact-demo-forked-53hvhw?file=/src/App.js

**Comment by pratheeshpradeepkumars** [comment]
@melloware I tried a using above sandbox workaround, same alert issue persists.
![image](https://github.com/primefaces/primereact/assets/32036647/afc6b49a-f2b6-4763-934f-e1e4065ed12e)

**Comment by melloware** [comment]
I think your tool may have an issue the select is marked `aria-hidden` ?

```xml
<select name="app-header" tabindex="-1" aria-hidden="true"><option value="" selected="">Select a City</option></select>
```

**Comment by melloware** [comment]
Ahh yep its your tool: https://wave.webaim.org/help

# How does WAVE handle "hidden" elements?

> WAVE detects accessibility issues in all page elements, even those that are hidden using CSS, the hidden attribute, aria-hidden="true", and/or tabindex="-1". This is by design. In most cases, hidden elements are revealed to users at some point. We believe it's important that these potential end user accessibility issues in drop-down menus, tab panels, dialog boxes, carousels, and other "hidden" elements be identified.
> 
> In most cases the WAVE interface will provide an indication of icons that are relevant to hidden elements—the icon in the sidebar will be slightly dimmed and will present an informative tooltip. These elements may be examined by turning off the Styles option in the WAVE sidebar.
> 
> If WAVE has detected an error in an element that is hidden and that is never presented to the end user, or that is perhaps modified to address the error before being presented (something WAVE cannot detect), the WAVE error can be ignored - or you may instead choose to adapt the code to ensure the element is accessible in its hidden state.
> 
> In short, if we err on this point, we err on the side of users with disabilities. In other words, we have chosen to identify issues in hidden content that may be presented to users, rather than suppress these errors in cases where advanced development techniques have been implemented to potentially hide that content from users.
